test/langtools/jdk/javadoc/lib/javadoc/tester/JavadocTester.java

```diff
@@ -488,15 +489,21 @@ public void javadoc(String... args) {
             }
         });
 
-        if (exitCode == Exit.OK.code && Files.exists(outputDir)) {
-            if (automaticCheckLinks) {
-                checkLinks();
+        if (Files.exists(outputDir)) {
+            if (automaticCheckNoStacktrace) {
+                checkOutput(Output.STDERR, false, " at com.sun.");
```

Comment:
> What is the reason for assuming package `com.sun` would be present in the stack trace? I notice that it is not present in the stack trace reported for the bug [JDK-8287379](https://bugs.openjdk.org/browse/JDK-8287379) which was recently fixed.

That's one reason why I suggested test-driving the new check by retrofitting it in the four existing use cases. Note that doclet/testSnippetTag/SnippetTester.java:55 looks for matches of:

    Pattern.compile("\\s*at.*\\(.*\\.java:\\d+\\)")

I checked, and that pattern matched those exceptions from JDK-8287379. Maybe we could use that instead of hard-coded ` at com.sun`.

Reply:
> What is the reason for assuming package `com.sun` would be present in the stack trace? I notice that it is not present in the stack trace reported for the bug [JDK-8287379](https://bugs.openjdk.org/browse/JDK-8287379) which was recently fixed.

That's one reason why I suggested test-driving the new check by retrofitting it in the four existing use cases. Note that doclet/testSnippetTag/SnippetTester.java:55 looks for matches of:

    Pattern.compile("\\s*at.*\\(.*\\.java:\\d+\\)")

While that pattern matches those exceptions from JDK-8287379, it might be too broad for the new check.

Reply:
Ooops.  I was (incorrectly) thinking of the old days, `com.sun.tools.javadoc` and `com.sun.tools.doclets`.